WORKSPACE

```diff
@@ -39,7 +39,7 @@ python_configure(
 #    curl -L https://github.com/openxla/xla/archive/<git hash>.tar.gz | sha256sum
```

Comment:
Please also update the `data` in setup.py so that other dependencies can match openxla/xla https://github.com/pytorch/xla/pull/7087/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7L67

Reply:
Which exact date should it be?

Reply:
The date is used for locating the nightly libtpu whl. libtpu and openxla need to match. The data should be 1 day prior to the pinned xla commit. (e.g. the pinned xla commit in on May 21st, let's put `20240520` here)